src/introduction-to-unified.md

```diff
@@ -0,0 +1,214 @@
+## Introduction to unified
+
+* * *
+
+After reading this guide you will:
+
+*   Understand what unified does.
+*   Get a taste of the ecosystem.
+*   Know how it can be used.
+*   Know what parts (processors) you need for your (future) use case.
+*   Have a list of resources to continue learning or get started.
+
+* * *
+
+![](https://cdn-images-1.medium.com/max/2600/1*j0qPjM4niq2S5kH9So1xww.png)
```

Comment:
might be good to co-locate image with content, on GitHub.

Reply:
Definitely! Forgot to change that afterwards.